nilearn/_utils/testing.py

```diff
@@ -24,11 +28,11 @@
 from . import logger
 
 try:
-    from nose.tools import assert_raises_regexp
+    from nose.tools import assert_raises_regex
```

Comment:
Fixed. Also dealt with those PEP8 violations... you are correct: that IS very annoying :)